### PR TITLE
alliance: 1.0.5-0 in 'indigo/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by adrianohrl**
Increasing version of package(s) in repository `alliance` to `1.0.5-0`:

- upstream repository: https://github.com/adrianohrl/alliance.git
- release repository: https://github.com/adrianohrl/alliance-release.git
- distro file: `indigo/distribution.yaml`
- bloom version: `0.6.1`
- previous version for package: `null`

## alliance

```
* removed rqt_mrta run depend
* Contributors: adrianohrl
```

## alliance_msgs

- No changes

## rqt_alliance

- No changes

### CODE REVIEW

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -232,6 +232,22 @@ repositories:
...
+      - rqt_alliance
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/adrianohrl/alliance-release.git
+      version: 1.0.5-0
+    source:
+      test_pull_requests: true
+      type: git
+      url: https://github.com/adrianohrl/alliance.git
+      version: '1.05'
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: This needs to point to a valid branch name instead.

File: indigo/distribution.yaml [hunk 2]
```diff
@@ -232,6 +232,22 @@ repositories:
...
+      packages:
+      - alliance
+      - alliance_msgs
+      - rqt_alliance
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/adrianohrl/alliance-release.git
+      version: 1.0.5-0
+    source:
+      test_pull_requests: true
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

File: indigo/distribution.yaml (hunk 1 above)
Comment (adrianohrl) [reply indigo/distribution.yaml]: It should be, at line 249:

version: '1.0.5'

File: indigo/distribution.yaml (hunk 2 above)
Comment (adrianohrl) [reply indigo/distribution.yaml]: It is awaiting ros-pull-request-builder's response.

File: indigo/distribution.yaml (hunk 1 above)
Comment (dirk-thomas) [reply indigo/distribution.yaml]: That is a tag but it needs to be a branch name.

Comment (adrianohrl) [reply indigo/distribution.yaml]: master or indigo-devel

Comment (dirk-thomas) [reply indigo/distribution.yaml]: Whichever contains the code you want to be used for CI testing.

**Comment by adrianohrl** [comment]
Thanks for your attention, @dirk-thomas, and sorry for any inconvenience.

**Comment by dirk-thomas** [comment]
> sorry for any inconvenience.

No problem :+1:

**Pull request merged**